src/components/SiteNav.jsx

```diff
@@ -1,5 +1,5 @@
 function SiteNav() {
-  return <p>SiteNav</p>;
+  return <p>Navigate</p>;
```

Comment:
What's the purpose of this text change? Does it need to be changed?

Reply:
I believe I deleted the <p>SiteNav</p>  in the return section because I originally had some Register button code in there. But I eventually deleted the code for the Register button, so I wanted to put the simple <p>SiteNav</p> back in there.